tests/clientcmd/command.go

```diff
@@ -0,0 +1,255 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package clientcmd
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+
+	"kubevirt.io/client-go/log"
+	"kubevirt.io/kubevirt/tests/flags"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const (
+	commandPipeFailed    = "command pipe failed"
+	commandPipeFailedFmt = "command pipe failed: %v"
+
+	serverName = "--server"
+)
+
+func GetK8sCmdClient() string {
+	// use oc if it exists, otherwise use kubectl
+	if flags.KubeVirtOcPath != "" {
+		return "oc"
+	}
+
+	return "kubectl"
+}
+
+func SkipIfNoCmd(cmdName string) {
```

Comment:
Basically I just move all relevant ones to here, subject oriented.
I can move it another step, but are you sure it is good to group all skips under the same file? It also seems odd to me that it is under `checks` sub-package.

But I do not have any strong feeling about this, as long as it is out of `utils` I am good.

Reply:
This is the question that comes up when I look at this skip thing: Should we group skips under `checks` or under individual subject packages?

Reply:
Both options seem logical to me. I think we can group them for now as I don't think there is a lot of them & we expect to don't have more of them.

Reply:
These are added and removed all the time.
The network sig have them under their helper [package](https://github.com/kubevirt/kubevirt/blob/main/tests/libnet/skips.go).

If you do not mind so much, can we move it after this PR?

BTW, one advantage I can think of to keep these per subject is the collision domain between different sigs. If we keep subjects/sigs separated as much as possible, it will avoid collisions and keep owners clear. IMO it will make it easier to find things (but this may not be a strong argument if there are not a lot of them around).